service/task.go

```diff
@@ -811,6 +811,7 @@ func (uis *UIServer) testLog(w http.ResponseWriter, r *http.Request) {
 		BaseURL:       uis.Settings.Cedar.BaseURL,
 		TaskID:        taskID,
 		TestName:      testName,
+		GroupID:       r.URL.Query().Get("group_id"),
```

Comment:
is GroupID available in `gimlet.GetVars(r)`

Reply:
No, `gimlet.GetVars(r)` only returns variables from the URL not query params.